features/FEATURE_BLE/ble/gap/Gap.h

```diff
@@ -279,6 +279,21 @@ class Gap : public StaticInterface<Impl, Gap> {
 #endif
     using StaticInterface<Impl, ::ble::interface::Gap>::impl;
 
+    /**
+     * Gap shutdown event handler.
+     *
+     * @see Gap::onShutdown().
+     */
+    typedef FunctionPointerWithContext<const Gap *> GapShutdownCallback_t;
```

Comment:
Note: This is not in the EventHandler, to align with the bahaviour of Gatt, SecurityManager, etc.
(And the resetting of Gap causes EventHandler pointer to be reset)